cabal-install/changelog

```diff
@@ -1,6 +1,9 @@
 -*-change-log-*-
 
 3.2.0.0 Herbert Valerio Riedel <hvr@gnu.org> April 2020
+3.2.0.0 Someone <mailto:somewhere@example.com> February 2020
```

Comment:
Add fine to `changelog.d` directory, see the `issue-*` files for an examples

It's recent change, so we won't introduce conflicts to changelog files.